apps/frontend/core-dapp/src/stores/entities/MarketEntity.ts

```diff
@@ -5,6 +5,7 @@ import { action, makeObservable, observable, reaction, runInAction } from 'mobx'
 import { IconName } from 'prepo-ui'
 import { ContractReturn, ContractStore, Factory } from 'prepo-stores'
 import { SEC_IN_MS } from 'prepo-constants'
+import { makeError } from 'prepo-utils/lib/src'
```

Comment:
You might wanna import from `prepo-utils`

```suggestion
import { makeError } from 'prepo-utils'
```

Reply:
Thanks! I need to see if I can stop my IDE from auto importing from there...